js/TutorModel.js

```diff
@@ -14,11 +14,15 @@ export default class TutorModel extends Backbone.Model {
       },
       ...Adapt.course.get('_tutor')
     };
+
+    defaults.altTitle ??= Adapt.course.get('_globals')._extensions._tutor.altTitle;
+    return defaults;
   }
 
   initialize(data, parentModel) {
     data = $.extend(true, this.defaults(), data?._isInherited === true ? null : data, {
       _attributes: { 'data-adapt-id': parentModel.get('_id') },
+      altTitle: parentModel.get('altFeedbackTitle'),
```

Comment:
This will always override the default unless it's undefined, which it won't be when you change the core. It'll be an empty string.